_posts/elements/2015-04-09-buttons.md

```diff
@@ -0,0 +1,105 @@
+---
+layout:       element
+title:        Buttons
+date:         2015-04-09 00:00:00
+
+categories:   elements
+tags:
+- atomic
+- pattern
+- buttons
+
+slug:         buttons
+scssurl:      "https://github.com/edx/ux-pattern-library/blob/master/_src/static/sass/components/_buttons.scss"
+
+description:  A collection of buttons available for using within the edX platform.
+
+info: Buttons should be used for performing actions within the edX environment. While we supply a button that looks like a link, it should only really be used for very tertiary _actions_. edX offers three button sizes, each with normal, hover, active/pressed, and disabled states. We also offer a range of other buttons to suit your needs.
```

Comment:
NIT: I don't think this paragraph is being parsed for markdown so the ``_actions_`` formatting isn't being applied. Mind removing it?